src/test/java/org/jabref/gui/journals/JournalAbbreviationsViewModelMixedAbbreviationsTabTest.java

```diff
@@ -69,8 +69,8 @@ void setUpViewModel(@TempDir Path tempFolder) throws Exception {
 
     @Test
     void testInitialHasNoFilesAndNoAbbreviations() {
-        assertEquals(0, viewModel.journalFilesProperty().size());
-        assertEquals(0, viewModel.abbreviationsProperty().size());
+        assertEquals(0, viewModel.journalFilesProperty().size(), "Incorrect setup of test. Journal files not set to 0 at setup.");
+        assertEquals(0, viewModel.abbreviationsProperty().size(), "Incorrect setup of test. Abbreviations not set to 0 at setup");
```

Comment:
Output seems reasonable here, if multiple asserts happen. On the other hand, why are there multiple asserts in one test? Should this be distinguished? Or is this trivial? @JabRef/developers